docs/Chromaticqa-Guide.md

```diff
@@ -0,0 +1,20 @@
+# ChromaticQA
+
+## Introduction to ChromaticQA
+[Chromatic Docs](https://docs.chromaticqa.com/)
+
+ChromaticQA is a visual regression tool for storybook maintained by Storybook.The purpose of having chromaticQA run on all stories is to capture unwanted visual changes introduced by every change done to the code base.This reduces the overhead on manual regression as it supports screenshot comparisons for chrome,IE and FF.
+
+## Running Test
+ChromaticQA runs as a part of the build.The builds can be seen here (https://www.chromaticqa.com/builds?appId=5d28eb3fe163f6002046d6fa)
+
+To run chromatic locally export the app code in bash profile(`export CHROMATIC_APP_CODE=<app-code>`) and run `npm run test:chromatic`
```

Comment:
hmm `npm run test:chromatic`has been removed because of this issue: https://github.com/bbc/simorgh/pull/3374

we can update the doc here to reflect that with 
```suggestion
To run chromatic locally export the app code in bash profile(`export CHROMATIC_APP_CODE=<app-code>`) and run `npx chromatic test run  --build-script-name build:storybook || true`
```